wire-ios/Wire-iOS/Sources/UserInterface/GroupDetails/ConversationActions/ConversationCallController.swift

```diff
@@ -52,10 +52,24 @@ final class ConversationCallController: NSObject {
         }
     }
 
-    func startVideoCall(started: (() -> Void)?) {
-        conversation.confirmJoiningCallIfNeeded(alertPresenter: target) { [conversation] in
-            started?()
-            conversation.startVideoCall()
+    func startVideoCall(started: Completion?) {
+        let startVideoCall = { [weak self] in
+            guard let `self` = self else { return }
+            self.conversation.confirmJoiningCallIfNeeded(alertPresenter: self.target) {
+                started?()
+                self.conversation.startVideoCall()
+            }
+        }
+
+        if conversation.localParticipants.count <= 4 {
```

Comment:
use a constant, does this number come from some config too?

Reply:
That number came from the product team as well. What would be a good name for that constant?

Reply:
what about ConfirmGroupCallParticipantsLimit ?

Reply:
Sounds good. I'm going to use that name for the property.

Reply:
Done!